File: java/src/main/java/vsdk/toolkit/raycasting/bidirectionalRaytracing/SparConfig.java
package vsdk.toolkit.raycasting.bidirectionalRaytracing;

// Spar Config stores handy config params
public class SparConfig {
    public BidirectionalPathRaytracerConfig baseConfig;

    // Needed in weighted multi-pass methods
    public Spar leSpar;
    public Spar ldSpar;
}

File: typescript/src/main/vsdk/toolkit/raycasting/bidirectionalRaytracing/SparConfig.ts
import { BidirectionalPathRaytracerConfig } from "./BidirectionalPathRaytracerConfig";
import { Spar } from "./Spar";

export class SparConfig {
  public baseConfig: BidirectionalPathRaytracerConfig | null;
  public leSpar: Spar | null;
  public ldSpar: Spar | null;

  public constructor() {
    this.baseConfig = null;
    this.leSpar = null;
    this.ldSpar = null;
  }
}
